### PR TITLE
Refactor downloader 2

### PR DESCRIPTION
**Pull request by apeksharma**
**Detailed description**:
Remove last pieces of duplicate code.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: src/main/java/com/hedera/mirror/downloader/Downloader.java [hunk 1]
```diff
@@ -64,53 +64,49 @@
...
-	protected final DownloaderProperties downloaderProperties;
-	private final ExecutorService signatureDownloadThreadPool; // Thread pool used one per node during the download process for signatures.
-
-	public Downloader(TransferManager transferManager, ApplicationStatusRepository applicationStatusRepository, NetworkAddressBook networkAddressBook, DownloaderProperties downloaderProperties) {
+    private final NetworkAddressBook networkAddressBook;
+	private final DownloaderProperties downloaderProperties;
+    // Thread pool used one per node during the download process for signatures.
+	private final ExecutorService signatureDownloadThreadPool;
+
+    private final Comparator<String> s3KeyComparator = (String o1, String o2) -> {
```
Comment (steven-sheehy) [inline src/main/java/com/hedera/mirror/downloader/Downloader.java]: I think it's better if the construction of this moves back into the constructor to keep declarations and instantiations separate.

File: src/main/java/com/hedera/mirror/downloader/Downloader.java [hunk 2]
```diff
@@ -64,53 +64,49 @@
 	protected final Logger log = LogManager.getLogger(getClass());
 
 	private final TransferManager transferManager;
-	private final Comparator<String> s3KeyComparator;
-	protected final Comparator<String> fileNameComparator;
-	protected final Comparator<Path> pathComparator;
-	protected List<String> nodeAccountIds;
+	private List<String> nodeAccountIds;
 	protected final ApplicationStatusRepository applicationStatusRepository;
```
Comment (steven-sheehy) [inline src/main/java/com/hedera/mirror/downloader/Downloader.java]: private

Comment (apeksharma) [reply src/main/java/com/hedera/mirror/downloader/Downloader.java]: done.

File: src/main/java/com/hedera/mirror/downloader/Downloader.java (hunk 1 above)
Comment (apeksharma) [reply src/main/java/com/hedera/mirror/downloader/Downloader.java]: done.

**Pull request merged**